deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -69,6 +72,10 @@ def compact_partition(params: CompactPartitionParams, **kwargs) -> Optional[str]
     assert (
         params.hash_bucket_count is not None and params.hash_bucket_count >= 1
     ), "hash_bucket_count is a required arg for compactor v2"
+    if params.num_rounds > 1:
+        assert (
+            not params.drop_duplicates
+        ), "num_rounds > 1, drop_duplicates must be False but is True"
```

Comment:
Do we need this assertion here? For backfill case in the future, the `drop_duplicates` will be True.

Reply:
That makes sense, do you think I should remove it?

Reply:
In my mind, I don't see a correlation here that if  `num_rounds > 1`, then `drop_duplicates` should be False. What's the initial thought behind adding this assertion?

Reply:
Oh I see. What I was thinking was that because multiple rounds just supports rebase for now, we should be making sure that we aren't trying to run multiple rounds with backfill (drop_duplicates being True)